src/Hyperwallet/Util/ApiClient.php

```diff
@@ -159,6 +160,7 @@ private function doRequest($method, $url, array $urlParams, array $options) {
             if ($response->getStatusCode() === 204) {
                 return array();
             }
+            $this->checkResponseHeaderContentType($response);
```

Comment:
In ApiClient.doRequest please add setting `Accept` http header to `application/jose+json` if the object reference is set to `isEncrypted = true` else set it to  `Accept` http header to `application/json`
Inside line 147~158

Reply:
Done